resources/lib/syncEpisodes.py

```diff
@@ -578,10 +575,10 @@ def __compareShows(self, shows_col1, shows_col2, rating=False, restrict=False):
 
                 if show_col2:
                     show = {'title': show_col1['title'], 'ids': {}, 'year': show_col1['year']}
-                    if 'tvdb' in show_col1['ids']:
-                        show['ids'] = {'tvdb': show_col1['ids']['tvdb']}
-                    if 'imdb' in show_col2 and show_col2['imdb']:
-                        show['ids']['imdb'] = show_col2['imdb']
+                    if show_col1['ids']:
+                        show['ids'].update(show_col1['ids'])
+                    if show_col2['ids']:
+                        show['ids'].update(show_col2['ids'])
```

Comment:
What I'm wondering right now is. If col1 has a tvdb id is it possible that it has a better tvdb id then col2? we would be using the col2 id now and possibly run into errors.